components/api-server/src/bin/api_server.rs

```diff
@@ -0,0 +1,126 @@
+use anyhow::Context;
+use axum::{
+    Json,
+    Router,
+    extract::{Path, State},
+    http::StatusCode,
+    response::{
+        IntoResponse,
+        Sse,
+        sse::{Event, KeepAlive},
+    },
+    routing::{get, post},
+};
+use clap::Parser;
+use clp_rust_utils::{clp_config::package, serde::yaml};
+use futures::{Stream, StreamExt};
+use thiserror::Error;
+use tracing_subscriber::{self, prelude::*};
+
+#[derive(Parser)]
+#[command(version, about = "API Server for CLP.")]
+struct Args {}
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    tracing_subscriber::registry()
+        .with(tracing_subscriber::fmt::layer())
+        .with(tracing_subscriber::EnvFilter::from_default_env())
+        .init();
+
+    let _ = Args::parse();
+    let home = std::env::var("CLP_HOME").context("Expect a CLP_HOME env variable")?;
+    let home = std::path::Path::new(&home);
+
+    let config_path = home.join(package::DEFAULT_CONFIG_FILE_PATH);
+    let config: package::config::Config = yaml::from_path(&config_path).context(format!(
+        "Config file {} does not exist",
+        config_path.display()
+    ))?;
+
+    let credentials_path = home.join(package::DEFAULT_CREDENTIALS_FILE_PATH);
+    let credentials: package::credentials::Credentials = yaml::from_path(&credentials_path)
+        .context(format!(
+            "Credentials file {} does not exist",
+            credentials_path.display()
+        ))?;
+
+    let addr = format!("{}:{}", &config.api_server.host, &config.api_server.port);
+    let listener = tokio::net::TcpListener::bind(&addr)
+        .await
+        .context(format!("Cannot listen to {addr}"))?;
+
+    let client = clp_client::Client::connect(&config, &credentials)
+        .await
+        .context("Cannot connect to CLP")?;
+
+    let app = Router::new()
+        .route("/", get(health))
+        .route("/health", get(health))
+        .route("/query", post(query))
+        .route("/query_results/{search_job_id}", get(query_results))
+        .with_state(client);
+
+    tracing::event!(tracing::Level::INFO, "Server started at {addr}");
```

Comment:
```suggestion
    tracing::info!("Server started at {addr}");
```
How about using `info!` directly?